archey/test/test_archey_logos.py

```diff
@@ -0,0 +1,50 @@
+"""Test module for `archey.logos`"""
+
+import unittest
+
+from archey.distributions import Distributions
+from archey.constants import LOGOS_DICT
+from archey.logos import get_logo_width
+
+
+class TestLogosModule(unittest.TestCase):
+    """Simple tests checking logos consistency and utility function logic"""
+    def test_distribution_logos_width(self):
```

Comment:
This will bring us some guarantees when users will add support for new distributions :ok_hand: